src/libraries/System.Private.CoreLib/src/System/Numerics/Quaternion.cs

```diff
@@ -51,6 +51,13 @@ public Quaternion(Vector3 vectorPart, float scalarPart)
             W = scalarPart;
         }
 
+        /// <summary>Gets a quaternion that represents a zero.</summary>
+        /// <value>A quaternion whose values are <c>(0, 0, 0, 0)</c>.</value>
+        public static Quaternion Zero
+        {
+            get => default;
```

Comment:
the API approved had `new()` here. as I understand it, that comes to the same thing as `default` today, but if a parameterless constructor was added in the future, it would begin to compile against that, whatever it does. So I think `default` is more correct, but someone else should confirm I'm not missing anything.

Reply:
Fixed.